bundles/core/pom.xml

```diff
@@ -141,6 +141,7 @@
                                     <goal>check</goal>
                                 </goals>
                                 <configuration>
+                                    <skip>true</skip>
```

Comment:
Can you please remove this config so it's not getting accidentally merged? You can also skip the check with the parameter `-Djacoco.skip=true` during development.